packages/core/src/context/message-context.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC, useEffect } from 'react'
+import {
+  MessageProvider,
+  useMessageController,
+  useMessages,
+} from './experiment'
+import { render, screen } from '@testing-library/react'
+
+describe('MessageProvider', () => {
+  const Tester: FC = () => {
```

Comment:
Why is this FC type annotation needed? (explicit variable typings should generally be avoided)